<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pret</class>
 <widget class="QWidget" name="pret">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Agence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="agence_list">
        <item>
         <property name="text">
          <string>Toutes Agences</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mulhouse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marseille</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paris</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Strasbourg</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Banquier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="banquier_list">
        <item>
         <property name="text">
          <string>Tous </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ahmed Benmoussa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alexandre Maurice</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alice Benarfa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yanis Moujab</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="date_picker"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <widget class="QLabel" name="pdp">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="username">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ALEX MOOR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>211</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,1,4">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QLabel" name="nom_agence">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Agence</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nom_banquier">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Banquier</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="photo">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align='center'&gt;&lt;img src=':/assets/logo.png'/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QtCharts::QChartView" name="chart_pret">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>571</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="vol_pret">
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/vol.png</normaloff>:/assets/vol.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_pret">
      <property name="text">
       <string>CA</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/ca.png</normaloff>:/assets/ca.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pourcentage_pret">
      <property name="text">
       <string>Pourcentage</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/pourc.png</normaloff>:/assets/pourc.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>chart_pret</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_4</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qtcharts/qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
